<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeamCarving</class>
 <widget class="QDialog" name="SeamCarving">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>415</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_num_cols_del">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_cols_del">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Cols to delete     : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_cols">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_cols_insert">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Cols to insert     : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>81</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Cols in Image   : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_rem">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw to Save or Remove Objects</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>51</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Rows in Image : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_rows_del">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Rows to delete   : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_rows">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num_rows_insert">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>239</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num_rows_del">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>178</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_rows_insert">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Rows to insert   : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num_cols_insert">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>165</width>
     <height>20</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SeamCarving</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SeamCarving</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
